src/devices/lcec_omrg5.c

```diff
@@ -220,6 +249,13 @@ static int lcec_omrg5_init(int comp_id, lcec_slave_t *slave) {
   hal_data = LCEC_HAL_ALLOCATE(lcec_omrg5_data_t);
   slave->hal_data = hal_data;
 
+  // check for deprecated type names
+  if(strstr(slave->type_name, "OmrG5_") != NULL) {
+    char new_type_name[16];
+    strncpy(new_type_name, slave->type_name+6, strlen(slave->type_name));
+    rtapi_print_msg(RTAPI_MSG_ERR, LCEC_MSG_PFX "The used type name '%s' is deprectaed. Please change the type name to 'R88D-%s-ECT'\n", slave->type_name, new_type_name);
```

Comment:
s/deprectaed/deprecated/ :-)

Also consider changing "Please change the type name" to "Please change the type name in your XML config file".  It'll probably be easier for new users.

Reply:
Done